### ISSUE TITLE
Adding a new log level with a new color

### ISSUE THREAD

**Issue by bartekpacia** [lmittmann/tint#71]
Hey, thanks a ton for this great module.

I'd like to add a new `LevelTrace` to logger I use in my project, but at the same time, would like to preserve coloring.

I found [an example of custom log levels](https://tip.golang.org/src/log/slog/example_custom_levels_test.go) being done with `log/slog`

Is it possible? I'd appreciate an example :) Thanks for all the hard work!

**Comment by bartekpacia** [comment]
Alternatively, is it possible to change the `DBG` label to something different, e.g. `TRC` (for trace)?

**Comment by lmittmann** [comment]
Yes, you can customize the log level via `ReplaceAttr` but you also need to handle colors. A simpler way to customize log levels is a popular request, that is tracked in #61.
